x-pack/libs/es-opensaml-security-api/src/main/java/org/opensaml/security/config/org/elasticsearch/xpack/security/authc/saml/OpenSamlXpackSecurityConfigurationPropertiesSource.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
```

Comment:
Given the simplicity of this file, I'd suggest we just put it under the apache license so the whole shadow-JAR file is apache licensed rather than being a weird mix.

Reply:
Good point, haven't thought about that, pushed [a76db0f](https://github.com/elastic/elasticsearch/pull/98199/commits/a76db0f53151cfb6d56a5865826b5af415074af7)